### DISCUSSION TITLE
Where does LV2 log output go?

### DISCUSSION THREAD

**Discussion by Philipp91** [falkTX/Carla#1420] [Q&A] [answered]
Hi,

When I load the lv2-test-plugin into Carla, where can I see log output like [this](https://git.kx.studio/falkTX/lv2-state-test/src/branch/master/lv2-state-test.c#L69)?

It doesn't automatically appear on Carla's stdout, but maybe I just need to tune some settings? Or look elsewhere?

Thanks
Philipp

**Comment by falkTX** [comment, marked as answer]
The logs tab?

If you are on windows though you need to run carla inside cmd.exe or powershell

**Reply by Philipp91** [reply]
:facepalm: thanks! Hadn't seen that tab and so I always looked on the terminal.
